.gitignore

```diff
@@ -1,9 +1,71 @@
-/node_modules
-/.grunt
-/.npm-debug.log
-/vendor
-/.sass-cache
-/static/**/*.map
-/static/img/**/*.bak
-/.php_cs.cache
-/.idea/
\ No newline at end of file
+## IMPORTANT ##
```

Comment:
I'm not sure that these two really belong in the same commit, but I'm not sure it's worth rebasing just for that.